Apply the hash value according to the `BlockHead` change

MichaelKim20 opened the pull request:
Slashing information was recently added to Agora's Block Header.
I apply the change to Stoa for this.

The PR(https://github.com/bpfkorea/boa-sdk-ts/pull/79) of Stoa must be merged first.

Relates to #234

TrustHenry commented on package-lock.json:
```diff
@@ -1,8 +1,3707 @@
   "name": "stoa",
   "version": "0.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```
"lockfileVersion": 2,   :(

TrustHenry replied:
we will reflect the SDK version first.

MichaelKim20 commented:
I extracted block data from Agora with JSON. And we provide it through the link below.
https://gist.github.com/MichaelKim20/be432a180793e7a901801b4d98ac5848
I changed the test code based on this.
